crates/bevy_ecs/src/schedule/parallel_executor.rs

```diff
@@ -531,20 +570,20 @@ mod tests {
 
         fn write_u64(counter: Res<Counter>, _query: Query<&mut u64>) {
             let mut count = counter.count.lock();
-            assert_eq!(*count, 4, "should always be the 5th system to run");
+            //assert_eq!(*count, 4, "should always be the 5th system to run");
```

Comment:
Is it intended?

Reply:
yes, most of those assertions that check count aren't correct. It would probably be better to check a bitfield or a set of str. (there is a TODO in the code for it)